dev/quickinstance/main.tf

```diff
@@ -0,0 +1,70 @@
+# module "base" {

```

Comment:
Next Step: Please fill these parameters
	After fill and commit, resolve this comment and merge the pull request